x/params/client/cli/tx.go

```diff
@@ -44,7 +44,7 @@ where proposal.json contains:
     {
       "subspace": "staking",
       "key": "MaxValidators",
-      "value": "105"
+      "value": 105
```

Comment:
Does this break amino encoding?

Reply:
No, in this case `MaxValidators` is a `uint16` which isn't string-encoded -- it's just an int.